__tests__/__snapshots__/kitchen-sink-test.tsx.snap

```diff
@@ -8160,7 +8166,10 @@ exports[`Kitchen sink snapshot Renders correctly and snapShot matches 1`] = `
             3
           </Text>
           <Text
+            allowFontScaling={true}
```

Comment:
Should we have these properties (allowFontScaling, maxFontSizeMultiplier, and minmumFontScale) use the values from the constants file instead of being hardcoded to true, 100, and 0.1? Or is the intent here to make sure that we see the difference between the components from our library and the components from RNP?

Reply:
Yes, we are using the constants defined, this is a snapshot, not the component. This is taking the values set for those variables. If we change the values and run yarn test -u , then it should update the snapshot again

Reply:
Good call...I missed that this was just the snapshot.